my_pyscf/mcscf/lasscf_rdm.py

```diff
@@ -503,6 +525,116 @@ def kernel(self, mo_coeff=None, casdm1frs=None, casdm2fr=None, conv_tol_grad=Non
 
         return self.e_tot, self.e_cas, self.casdm1frs, self.casdm2fr, self.mo_coeff, self.mo_energy, h2eff_sub, veff
 
+class extremeAsynLASSCF (LASSCFNoSymm):
```

Comment:
At a certain point, if this gets too big, it would make sense to put it in a new file.